sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -644,6 +657,12 @@ def _create_test_data():
       data.append(("key", scientists[index]))
     return data
 
+  class _ExpandIterable(DoFn):
```

Comment:
_ExpandValuesIterable? 

This could also be `FlatMapTuple(lambda k, vs: vs)`

Reply:
Good to know! Replaced with `FlatMapTuple` :D